src/test/java/com/commercetools/project/sync/customobject/CustomObjectSyncerTest.java

```diff
@@ -0,0 +1,82 @@
+package com.commercetools.project.sync.customobject;
+
+import static com.commercetools.project.sync.util.TestUtils.getMockedClock;
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.commercetools.sync.customobjects.CustomObjectSync;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.customobjects.CustomObjectDraft;
+import io.sphere.sdk.customobjects.queries.CustomObjectQuery;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import org.junit.jupiter.api.Test;
+
+public class CustomObjectSyncerTest {
+
+  @Test
+  void of_ShouldCreateCustomObjectSyncerInstance() {
+    // test
+    final CustomObjectSyncer customObjectSyncer =
+        CustomObjectSyncer.of(mock(SphereClient.class), mock(SphereClient.class), getMockedClock());
+
+    // assertions
+    assertThat(customObjectSyncer).isNotNull();
+    assertThat(customObjectSyncer.getQuery()).isEqualTo(CustomObjectQuery.of(JsonNode.class));
+    assertThat(customObjectSyncer.getSync()).isExactlyInstanceOf(CustomObjectSync.class);
+  }
+
+  @Test
+  void transform_ShouldConvertResourcesToDrafts() {
+    // preparation
+    final CustomObjectSyncer customObjectSyncer =
+        CustomObjectSyncer.of(mock(SphereClient.class), mock(SphereClient.class), getMockedClock());
+
+    final CustomObject<JsonNode> customObject1 = mock(CustomObject.class);
+    when(customObject1.getContainer()).thenReturn("testContainer1");
+    when(customObject1.getKey()).thenReturn("testKey1");
+    when(customObject1.getValue()).thenReturn(JsonNodeFactory.instance.textNode("testValue1"));
+
+    final CustomObject<JsonNode> customObject2 = mock(CustomObject.class);
+    when(customObject2.getContainer()).thenReturn("testContainer2");
+    when(customObject2.getKey()).thenReturn("testKey2");
+    when(customObject2.getValue()).thenReturn(JsonNodeFactory.instance.booleanNode(true));
+    final List<CustomObject<JsonNode>> drafts = asList(customObject1, customObject2);
```

Comment:
Can it be changed to another name? Because they are not `CustomObjectDraft`

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/167/commits/d015858e9467cf0cc68e6dd1387a200b85ba1cdc